internal/ssm_test.go

```diff
@@ -31,13 +34,27 @@ func TestGetSSMParameter(t *testing.T) {
 	// Setup Test
 	mockSvc := &mockSSMClient{}
 
-	for k, v := range ssmTestValues {
-		t.Logf("Key: %s should have value: %s", k, v.expectedValue)
-		value, err := getSSMParameter(mockSvc, v.path, v.required, v.decrypt)
-		if err != nil {
-			t.Errorf("Expected %s , got %s", v.expectedValue, err)
-		} else if value != v.expectedValue {
-			t.Errorf("Expected %s , got %s", v.expectedValue, value)
+	for k, v := range fakeSSMStore {
+		expectedValueStr := "nil"
+		if v.expectedValue != nil {
+			expectedValueStr = *v.expectedValue
+		}
+		t.Logf("Key: %s should have value: %s", k, expectedValueStr)
+
+		value, err := getSSMParameter(mockSvc, k, v.defaultValue, false)
+
+		if v.expectedValue != nil && value != nil && *value == *v.expectedValue {
```

Comment:
`&& err != nil`
so you can use "reflect" `DeepEqual`
https://golang.org/pkg/reflect/#DeepEqual

Reply:
used assert for this